M2/Macaulay2/m2/html.m2

```diff
@@ -30,7 +30,7 @@ html Hypertext := x -> (
     T := class x;
     qname := T.qname;
     attr := "";
-    cont := if member(Options, keys T) then (
```

Comment:
Was this a bug?

Reply:
not a bug. I must've included this unintentionally. anyway, `T.?Options` is the standard way to check for the existence of the key `Options`.